### DISCUSSION TITLE
关于为什么不内置数据的说明

### DISCUSSION THREAD

**Discussion by hefengbao** [hefengbao/jingmo#44] [Announcements]
1. 诗文数据  110 万，大小1.3 G， 即便压缩后也有 350 M，app 打包后差不多 400 M;
2. Gitee  最大能上传的附件50 M，Github 限制 2 G 以内，但是国内有多少人能顺顺利利从 Github 下载 400 M 的文件？
3. 应用更新频繁，但是数据基本是固定的，每次下载那么大的包有必要吗？
4. 目前的数据用 Github Page 搭建， github.io 应该大多数人都能访问到才对。
5. 虽然同步数据要花时间，但是以后的 app 更新会很方便。

因而我认为数据和 app 分离是最优解！

**Comment by 2980077544** [comment]
希望可以支持用外部下载器下载，然后在应用内导入。像1DM之类的应用可以提高下载速度

**Comment by youcaidaxigua** [comment]
基本同步不下来 也不支持断点续传 问题太大了，可以选择让大家下载离线版或者在线版的。
